Makefile

```diff
@@ -1,6 +1,7 @@
 GOARCH?=$(shell go env GOARCH)
 GOOS?=$(shell uname -s | tr A-Z a-z)
 GOLANGCI_VERSION?=v1.24.0
+INTERNAL=github.com/rhobs/kube-events-exporter/internal
```

Comment:
nit: usually it is more common to put only repo name into a variable, especially that you are using this variable only once.